runtime/ops/http.rs

```diff
@@ -160,15 +161,16 @@ async fn op_http_request_next(
   let cancel = RcRef::map(conn_resource.clone(), |r| &r.cancel);
 
   poll_fn(|cx| {
+    conn_resource.deno_service.waker.register(cx.waker());
     let connection_closed = match conn_resource.poll(cx) {
       Poll::Pending => false,
       Poll::Ready(Ok(())) => {
-        // close ConnResource
-        state
+        // try to close ConnResource, but don't unwrap as it might
```

Comment:
If the conn could already be closed, doesn't this run the risk of closing the wrong resource?

Reply:
Not really - `ConnResources` might have been taken by one of the other ops and we don't use `rid`s so there's no risk that you close random resource by mistake.